icons/hicolor/convert.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+for x in 16 22 32 48 64 256
+do
+  mkdir -p ${x}x${x}/apps
+  for name in `ls scalable/apps/ | grep ".svg" | sed 's/.svg//g'`
+  do
+    inkscape -z --export-png ${x}x${x}/apps/${name}.png scalable/apps/${name}.svg -w ${x} 2> /dev/null 1> /dev/null
```

Comment:
Throwing away all Inkscape output? I tried canceling that and got some interesting errors:

```
scalable/apps/yast-license.svg:10: namespace error : xmlns:sodipodi: 'http://inkscape.sourceforge.net/DTD/s odipodi-0.dtd' is not a valid URI
   xmlns:sodipodi="http://inkscape.sourceforge.net/DTD/s odipodi-0.dtd"
...
** (inkscape:31476): WARNING **: No cc:license metadata found, derived license URI from cc:License: http://creativecommons.org/publicdomain/zero/1.0/
Bitmap saved as: 16x16/apps/yast-iucvterminal-server.png
```

Reply:
Unfortunately older icons tend to have a little mangled SVG, because instead of being saved as clean SVG, they are saved as Inkscape project SVGs

Reply:
There are two issues:
1. Inkscape reports errors. OK, we may be able to address their causes or not, I guess that the new-icons list works around your comment above.
2. The errors are thrown away with `2>/dev/null`. Please *don't* do that. (`1>/dev/null` is fine in this case) If you remove the `2>` redirect you will find a problem with the new-icons list. Please fix it.